.github/workflows/docker-connector-image-publishing.yml

```diff
@@ -70,15 +69,18 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v3
 
-      - name: Set image build variables
+      - name: Set and debug variables
         id: vars
         run: |
           DOCKERFILE=docker-images/Dockerfile.${{ github.event.inputs.connector-type }}-connector-base
           IMAGE_NAME=${{ github.event.inputs.connector-type }}-connector-base
-          FULL_IMAGE_REF=${{ github.event.inputs.repository-root }}${IMAGE_NAME}:${{ github.event.inputs.tag-or-version-number }}
+          FULL_IMAGE_REF=${{ github.event.inputs.repository-root }}/${IMAGE_NAME}:${{ github.event.inputs.tag-or-version-number }}
```

Comment:
is the additional slash here the main fix?

Reply:
@davinchia - That, plus removing registry from the docker login.

Reply:
Oh - and weirdly the stringy comparison was needed, in place of the boolean-type comparison.

<img width="502" alt="image" src="https://github.com/user-attachments/assets/7df52bea-cc19-4615-9fe9-de8cb964ff56" />